src/deadline/client/api/_submit_job_bundle.py

```diff
@@ -418,10 +421,15 @@ def _default_update_upload_progress(upload_metadata: Dict[str, str]) -> bool:
     if not upload_progress_callback:
         upload_progress_callback = _default_update_upload_progress
 
+    # Create job history dir for the manifest file(s)
```

Comment:
This creates a directory in the history dir only for holding the manifests. I don't think that's quite what we want, it seems better to use a temp directory in this code path.

Reply:
I thought the job history dir would be a good common area for users to look in, but with Nathan's suggestion for a specific flag and printing the path for the CLI flow I think a temp dir works better.